docs/user_guides/fs/feature_view/helper-columns.md

```diff
@@ -11,17 +11,17 @@ Hopsworks Feature Store provides a functionality to define two types of helper c
 
 ## Inference Helper columns
 `inference_helper_columns` are a list of feature names that are not used for training the model itself but are used for extra information during online or batch inference. 
-For example computing [on-demand feature](../../../concepts/fs/feature_group/on_demand_feature.md) like distance between previous and current place of transaction `loc_delta_t_minus_1` in credit card fraud detection system.
-Feature `loc_delta_t_minus_1` will be computed using previous transaction coordinates `longitude` and `latitude` that needs to fetched from the feature store and compared to the new transaction coordinates that arrives at inference application. 
-In this use case `longitude` and `latitude` are `inference_helper_columns`. They are not used for training but are necessary for computing [on-demand feature](../../../concepts/fs/feature_group/on_demand_feature.md) `loc_delta_t_minus_1`.
+For example computing [on-demand feature](../../../concepts/fs/feature_group/on_demand_feature.md) like days left until credit card is valid at the time of transaction `days_until_valid` in credit card fraud detection system.
```

Comment:
For example, computing an on-demand feature such as `days_valid` (days left that a credit card is valid at the time of the transaction) in a credit card fraud detection system.

It is not until_valid. The card is valid, days until it is invalid. But better to have a positive. `days_valid`, rather than a negative - days_until_no_longer_valid